sdk/tests/deqp/functional/gles3/es3fShaderCommonFunctionTests.js

```diff
@@ -276,27 +276,34 @@ goog.scope(function() {
     es3fShaderCommonFunctionTests.CommonFunctionCase.prototype.iterate = function() {
         /** @type {number} */ var numInputScalars = es3fShaderCommonFunctionTests.computeTotalScalarSize(this.m_spec.inputs);
         /** @type {number} */ var numOutputScalars = es3fShaderCommonFunctionTests.computeTotalScalarSize(this.m_spec.outputs);
-        /** @type {goog.TypedArray} */ var inputData;
-        /** @type {goog.TypedArray} */ var outputData;
+        /** @type {Array<goog.TypedArray>} */ var inputData = [];
+        /** @type {Array<goog.TypedArray>} */ var outputData = [];
         /** @type {gluShaderUtil.DataType} */ var inputType = this.m_spec.inputs[0].varType.getBasicType();
         /** @type {gluShaderUtil.DataType} */ var outputType = this.m_spec.outputs[0].varType.getBasicType();
-        /** @type {Array<number>} */ var inputValues;
+        /** @type {Array<Array<number>>} */ var inputValues;
```

Comment:
Is this really necessary?  It looks to me getInputValues() always only return one array, never multiple arrays. So maybe Array<number> is OK? This could simply the code quite a bit.